doc/developer/plugins/customview.rst

```diff
@@ -84,7 +87,7 @@ Frontend views
 --------------
 
 Frontend views work pretty much like organiser area views. Take care that your URL starts
-with ``/(P?P<event>[\]+)/``, or even better, ``/(P?P<event>[\]+)/p/``.
+with ``f'^(?P<event>[{SLUG_CHARS}]+)/``
```

Comment:
We really want to encourage plugins to include `/p/` in their URL to reduce collisions, especially with future URLs, so I'd like to keep this part in (though with the proper `SLUG_CHARS` of course).

Reply:
Fixed.